### PR TITLE
[tr064] Update configuration documentation

### PR DESCRIPTION
**Pull request by lsiepel**
Signed-off-by: lsiepel <leosiepel@gmail.com>

- Improves documentation for comma separated list of MAC-Addresses 

Replaces: https://github.com/openhab/openhab-addons/pull/13988/
Fixes: https://github.com/openhab/openhab-addons/issues/13798

### CODE REVIEW

File: bundles/org.openhab.binding.tr064/src/main/resources/OH-INF/i18n/tr064.properties [hunk 1]
```diff
@@ -35,29 +35,32 @@ thing-type.config.tr064.fritzbox.rejectedCallDays.label = Rejected Call Days
...
 thing-type.config.tr064.subdeviceLan.refresh.label = Refresh Interval
 thing-type.config.tr064.subdeviceLan.uuid.label = UUID
 thing-type.config.tr064.subdeviceLan.uuid.description = UUID of the sub-device
 
 # thing types config
 
+thing-type.config.tr064.fritzbox.timeout.label = Timeout
+thing-type.config.tr064.fritzbox.timeout.description = Timeout for all requests (SOAP requests, phone book retrieval, call lists, ...).
+thing-type.config.tr064.generic.timeout.label = Timeout
+thing-type.config.tr064.generic.timeout.description = Timeout for all requests (SOAP requests, phone book retrieval, call lists, ...).
```
Comment (jlaur) [inline bundles/org.openhab.binding.tr064/src/main/resources/OH-INF/i18n/tr064.properties:60]: How did they end here? Perhaps no longer referenced?

**Review by jlaur — CHANGES_REQUESTED** [review]
Thanks! Seems like some I18N cleanup is needed.

File: bundles/org.openhab.binding.tr064/src/main/resources/OH-INF/i18n/tr064.properties (hunk 1 above)
Comment (lsiepel) [reply bundles/org.openhab.binding.tr064/src/main/resources/OH-INF/i18n/tr064.properties:60]: If they are no longer referenced, i would suspect that the i18n tool would remove them.
I only ran the i18n mvn command and this is the result. As far as i can see there is no real change, the lines are only moved to a different place in the file. So i would suggest to merge it as is. Please advice if you have other suggestions.

Comment (jlaur) [reply bundles/org.openhab.binding.tr064/src/main/resources/OH-INF/i18n/tr064.properties:60]: You can try to delete this section and re-run the tool. If they don't reappear, they are no longer referenced.

Comment (lsiepel) [reply bundles/org.openhab.binding.tr064/src/main/resources/OH-INF/i18n/tr064.properties]: Done. Seems there where more issues, i completely remove all contents and re-run the tool. It cleaned more entries.

File: bundles/org.openhab.binding.tr064/src/main/resources/OH-INF/i18n/tr064.properties [hunk 2]
```diff
@@ -35,46 +35,19 @@ thing-type.config.tr064.fritzbox.rejectedCallDays.label = Rejected Call Days
...
-phonebookLookupActionLabel = lookup a phone number
-phonebookLookupActionDescription = Lookup a phone number.
-phonebookLookupActionOutputLabel = Name
-phonebookLookupActionOutputDescription = The name associated with the phone number (or the phone number if no match can be found).
-phonebookLookupActionInputPhoneNumberLabel = Phone Number
-phonebookLookupActionInputPhoneNumberDescription = The phone number to be resolved.
-phonebookLookupActionInputPhoneBookLabel = Phone Book
-phonebookLookupActionInputPhoneBookDescription = The name of the the phone book.
-phonebookLookupActionInputMatchesLabel = Match Count
-phonebookLookupActionInputMatchesDescription = The number of digits matching the incoming value, counted from far right (default is 0 = all matching). Negative numbers skip digits from the left.
```
Comment (jlaur) [inline bundles/org.openhab.binding.tr064/src/main/resources/OH-INF/i18n/tr064.properties:80]: These are still referenced by `@text/` references here:
https://github.com/openhab/openhab-addons/blob/be6b799ca447b55ce69a068e65af57d3d843d11b/bundles/org.openhab.binding.tr064/src/main/java/org/openhab/binding/tr064/internal/phonebook/PhonebookActions.java#L42-L66

Comment (lsiepel) [reply bundles/org.openhab.binding.tr064/src/main/resources/OH-INF/i18n/tr064.properties:80]: I’ll have a look, probably revert everything as it was. 
This is something the i18n maven command does not pickup?

Comment (jlaur) [reply bundles/org.openhab.binding.tr064/src/main/resources/OH-INF/i18n/tr064.properties:80]: The tool only picks strings from the XML sources. For references like above, the I18N properties file **is** the source.

Comment (lsiepel) [reply bundles/org.openhab.binding.tr064/src/main/resources/OH-INF/i18n/tr064.properties]: Checked and fixed the actions. Thanks for the details. Still learning alot here ;-)

Comment (lsiepel) [reply bundles/org.openhab.binding.tr064/src/main/resources/OH-INF/i18n/tr064.properties]: I could not find any reference for the `profile-type.transform.xxx` but don't know where to expect those so i might be looking at the wrond place.

Comment (jlaur) [reply bundles/org.openhab.binding.tr064/src/main/resources/OH-INF/i18n/tr064.properties]: They have a suspicious duplicated header "# thing types config", so probably they are not relevant anymore. If you want to be 100% sure, you could check git history and find the commit where they moved, and check other changes in that commit.

Comment (lsiepel) [reply bundles/org.openhab.binding.tr064/src/main/resources/OH-INF/i18n/tr064.properties]: Seems like they are merged by https://github.com/openhab/openhab-addons/commit/f9695f2d8dd234bdba9b96a5c902b37798b9fded#diff-6114fe980cebbb1ca878651701ea434d443f2ed2071402e103f41683d5679349
and somehow referenced from [bundles/org.openhab.binding.tr064/src/main/resources/OH-INF/config/phonebookProfile.xml](https://github.com/openhab/openhab-addons/commit/f9695f2d8dd234bdba9b96a5c902b37798b9fded#diff-6114fe980cebbb1ca878651701ea434d443f2ed2071402e103f41683d5679349)

The uri in the xml is profile:transform:PHONEBOOK and in the translation is profile.config.transform.PHONEBOOK 

i'm to unfamiliair with this piece of translation / binding specifics, it also goes beyond the goal of this PR. So do you agree that i revert the removal of these lines and you merge it as is?

**Review by jlaur — APPROVED** [review]
LGTM

**Pull request merged**